app/views/layouts/application.html.erb

```diff
@@ -76,6 +76,7 @@
         </span>
         </p>
       </div>
+      <%= yield(:before_content) %>
```

Comment:
maybe we can be a bit more semantic here, which kind of content `before_content` is going to contain? this question should help to name it (in theory :P )

Reply:
I think `before_content` is fine, it hasn't caused confusion in `manage-courses-frontend`. It's a pretty standard Railsy thing.

Reply:
Could be a back link or breadcrumbs as they go above the `govuk-main-wrapper`. I stole the name from Find. 😈

What would you suggest?

Reply:
I like `breadcrumbs`, it sounds clearer for me, is it ok for you @tvararu? happy to leave as it is you have a strong opinion